utils/avd_start.sh

```diff
@@ -11,6 +11,7 @@ fi
 bash $PWD/utils/avd_stop.sh
 
 echo "Starting emulator"
+# nohup ${ANDROID_HOME}/emulator/emulator -avd android33 -gpu swiftshader &
```

Comment:
revert commented line?

Reply:
that's as a reminder that on linux i should use software compositor, otherwise headed mode kills xserver